pkg/query/api/v1.go

```diff
@@ -204,6 +204,17 @@ func (api *API) parseEnableDedupParam(r *http.Request) (enableDeduplication bool
 	return enableDeduplication, nil
 }
 
+func (api *API) parseReplicaLabelsParam(r *http.Request) (replicaLabels []string, _ *ApiError) {
+	if err := r.ParseForm(); err != nil {
+		return nil, &ApiError{ErrorInternal, errors.Wrap(err, "parse form")}
+	}
+
+	if len(r.Form["replicaLabels[]"]) > 0 {
+		replicaLabels = r.Form["match[]"]
```

Comment:
This doesn't seem quite right, is this a copy paste mistake? I would have expected `r.Form["replicaLabels[]"]`.

Reply:
good catch!, Thanks!